[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:329 



[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:330 Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.4.4    unsupported  01:49:18Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active       2  opensearch                                 0  no       
grafana-agent                      blocked      4  grafana-agent             latest/edge    251  no       Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main                               active       1  opensearch                                 2  no       
opensearch                         active       1  opensearch                                 1  no       
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent      Machine  Public address  Ports     Message
failover/0*                  active    executing  2        10.190.109.238  9200/tcp  
  grafana-agent/1            blocked   idle                10.190.109.238            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
failover/1                   active    executing  3        10.190.109.102  9200/tcp  
  grafana-agent/0*           blocked   idle                10.190.109.102            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main/0*                      active    executing  4        10.190.109.141  9200/tcp  (set-password) 
  grafana-agent/2            blocked   executing           10.190.109.141            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
opensearch/0*                active    executing  1        10.190.109.148  9200/tcp  
  grafana-agent/3            blocked   idle                10.190.109.148            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
self-signed-certificates/0*  active    idle       0        10.190.109.253            

Machine  State    Address         Inst id        Base          AZ  Message
0        started  10.190.109.253  juju-54ed64-0  ubuntu@22.04      Running
1        started  10.190.109.148  juju-54ed64-1  ubuntu@22.04      Running
2        started  10.190.109.238  juju-54ed64-2  ubuntu@22.04      Running
3        started  10.190.109.102  juju-54ed64-3  ubuntu@22.04      Running
4        started  10.190.109.141  juju-54ed64-4  ubuntu@22.04      Running

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:18 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [executing (since: 01:49:14)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:30 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [idle (since: 01:49:22)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:42 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [idle (since: 01:49:22)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:53 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [idle (since: 01:49:22)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:04 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [idle (since: 01:49:22)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:15 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [idle (since: 01:49:22)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:26 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.788* -- (10.190.109.141) -- [idle (since: 01:49:22)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:38 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.0d7* -- (10.190.109.148) -- [idle (since: 01:49:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:52 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:348 01:50:55 -- Waiting for model: complete.
[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.4.4    unsupported  01:50:56Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active       2  opensearch                                 0  no       
grafana-agent                      blocked      4  grafana-agent             latest/edge    251  no       Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main                               active       1  opensearch                                 2  no       
opensearch                         active       1  opensearch                                 1  no       
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0*                  active    idle   2        10.190.109.238  9200/tcp  
  grafana-agent/1            blocked   idle            10.190.109.238            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
failover/1                   active    idle   3        10.190.109.102  9200/tcp  
  grafana-agent/0*           blocked   idle            10.190.109.102            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main/0*                      active    idle   4        10.190.109.141  9200/tcp  
  grafana-agent/2            blocked   idle            10.190.109.141            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
opensearch/0*                active    idle   1        10.190.109.148  9200/tcp  
  grafana-agent/3            blocked   idle            10.190.109.148            Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
self-signed-certificates/0*  active    idle   0        10.190.109.253            

Machine  State    Address         Inst id        Base          AZ  Message
0        started  10.190.109.253  juju-54ed64-0  ubuntu@22.04      Running
1        started  10.190.109.148  juju-54ed64-1  ubuntu@22.04      Running
2        started  10.190.109.238  juju-54ed64-2  ubuntu@22.04      Running
3        started  10.190.109.102  juju-54ed64-3  ubuntu@22.04      Running
4        started  10.190.109.141  juju-54ed64-4  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-0: 01:40:46 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:40:46 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 01:40:46 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 01:41:22 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:41:22 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 01:41:23 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:41:24 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:41:24 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:41:24 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:41:24 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 01:41:24 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-failover-1: 01:41:24 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 01:41:27 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 01:41:27 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-0: 01:41:27 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-failover-0: 01:42:58 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:42:59 ERROR unit.failover/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-1: 01:42:59 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:00 ERROR unit.failover/1.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-0: 01:43:00 ERROR unit.failover/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-0: 01:43:01 ERROR unit.failover/0.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-failover-1: 01:43:01 ERROR unit.failover/1.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-main-0: 01:43:02 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:02 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-1: 01:43:02 ERROR unit.failover/1.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-0: 01:43:03 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:43:03 ERROR unit.main/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-1: 01:43:03 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-1: 01:43:04 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:43:04 ERROR unit.main/0.juju-log certificates:12: [Errno 111] Connection refused
unit-failover-0: 01:43:07 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:08 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:08 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 01:43:08 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:09 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-1: 01:43:09 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 01:43:09 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:10 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-opensearch-0: 01:43:10 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:43:10 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:10 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:11 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:11 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:11 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:11 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-main-0: 01:43:11 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:43:11 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:11 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:11 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:11 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:11 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-main-0: 01:43:11 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:11 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:43:11 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 01:43:12 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 01:43:12 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:12 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:43:12 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:12 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-1: 01:43:12 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:12 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:12 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-0: 01:43:13 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:43:13 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:14 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:43:14 ERROR unit.failover/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-1: 01:43:15 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-0: 01:43:15 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-main-0: 01:43:15 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:15 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:15 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:16 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-opensearch-0: 01:43:16 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:17 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:17 ERROR unit.opensearch/0.juju-log upgrade-version-a:2: [Errno 111] Connection refused
unit-failover-1: 01:43:18 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:43:18 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:18 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:18 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:19 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:19 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:20 ERROR unit.failover/0.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-opensearch-0: 01:43:20 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:21 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:43:21 ERROR unit.failover/1.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-main-0: 01:43:21 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:21 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:43:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:22 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:43:22 ERROR unit.failover/1.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-failover-0: 01:43:23 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:23 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:24 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:24 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:43:24 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-main-0: 01:43:24 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:24 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:43:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:25 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:43:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:43:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:25 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:25 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:43:25 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:26 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:26 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:26 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:26 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:26 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:27 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:43:27 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:43:27 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:28 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:43:29 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:43:30 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 01:43:30 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:43:30 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 01:43:31 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:32 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:33 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:43:34 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:35 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:37 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:43:37 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:43:38 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:43:58 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 01:43:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-failover-1: 01:44:03 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:03 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:03 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:03 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:04 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:04 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:04 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:04 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:05 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:44:05 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:44:05 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:44:05 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:44:06 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:44:06 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:06 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: Cannot connect to the OpenSearch server...
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 01:44:07 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:07 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:07 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 01:44:07 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:07 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:07 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:08 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:08 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:08 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:08 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 01:44:08 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:08 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:08 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:08 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:09 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:09 ERROR unit.failover/1.juju-log peer-cluster:9: Cannot connect to the OpenSearch server...
unit-failover-1: 01:44:09 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 01:44:09 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:44:09 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:44:09 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:44:09 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:44:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:44:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:44:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:44:10 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:10 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:10 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:10 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: Cannot connect to the OpenSearch server...
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:11 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:12 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:12 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:12 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:13 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 01:44:13 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:15 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:18 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:21 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 01:44:25 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:25 ERROR unit.failover/0.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-0: 01:44:25 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:26 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:26 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:26 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:26 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:26 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:26 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:26 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:26 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:28 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:28 ERROR unit.failover/0.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-0: 01:44:28 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:28 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:28 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:28 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:29 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:29 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:29 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:29 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:29 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:30 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 01:44:30 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:30 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:30 ERROR unit.failover/0.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-0: 01:44:30 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:33 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:36 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:39 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 01:44:42 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 01:44:47 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:44:54 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:44:55 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:44:55 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:44:56 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:44:56 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:44:58 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:58 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:58 ERROR unit.opensearch/0.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:44:58 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:58 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:58 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:59 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:45:00 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:45:00 ERROR unit.opensearch/0.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:45:00 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:45:03 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:45:06 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:45:09 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:45:12 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-grafana-agent-0: 01:46:50 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-251" for grafana-agent/0: failed to download charm "ch:amd64/jammy/grafana-agent-251" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-1: 01:46:50 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-251" for grafana-agent/1: failed to download charm "ch:amd64/jammy/grafana-agent-251" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-2: 01:46:50 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-251" for grafana-agent/2: failed to download charm "ch:amd64/jammy/grafana-agent-251" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-3: 01:46:50 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-251" for grafana-agent/3: failed to download charm "ch:amd64/jammy/grafana-agent-251" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-0: 01:46:50 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-1: 01:46:50 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-2: 01:46:50 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-3: 01:46:51 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...